packages/gatsby-source-wordpress/src/steps/preview/index.ts

```diff
@@ -296,12 +305,24 @@ export const sourcePreview = async (
     sendPreviewStatus,
   })
 
-  await fetchAndCreateSingleNode({
+  const { node } = await fetchAndCreateSingleNode({
     actionType: `PREVIEW`,
     ...previewData,
     previewParentId: previewData.parentDatabaseId,
     isPreview: true,
   })
+
+  const manifestId = node.databaseId + previewData.modified
+
+  reporter.info(
+    formatLogMessage(
+      `Creating node manifest for ${node.id} with manifestId ${manifestId}`
+    )
+  )
+  actions.unstable_createNodeManifest({
+    manifestId,
+    node,
+  })
```

Comment:
Should we be backward compatible when this API is not available? Only users with Gatsby 3.7.0 will be able to build now.

Reply:
Oh, yep definitely! Good call, I'll update it